backend/api/routers/hello.py

```diff
@@ -11,5 +12,9 @@
 
 
 @router.get('/')
```

Comment:
Why are you using hello router and not some more meaningful one?

Reply:
Cause that's the only one we've got

Reply:
How about you add some new one or change the name of this one?